module/support/NUsight/src/Vision.cpp

```diff
@@ -73,49 +74,15 @@ namespace support {
                 last_classified_image = NUClear::clock::now();
             }));
 
-        handles["vision_object"].push_back(
-            on<Trigger<std::vector<Ball>>, Single, Priority::LOW>().then([this](const std::vector<Ball>& balls) {
-                auto nusight = std::make_unique<NUsightBalls>();
+        handles["vision_object"].push_back(on<Trigger<Balls>, Single, Priority::LOW>().then([this](const Balls& balls) {
+            auto nusight = std::make_unique<Balls>();
+            emit<Scope::NETWORK>(nusight, "nusight", false);
```

Comment:
This is currently just sending an empty list of balls,
Also you should be doing the forwarding thing like from other places in this file.